src/minesweeper/utils/graphs/graphs.ts

```diff
@@ -0,0 +1,33 @@
+import { IPosition } from '@minesweeper/domain/models';
+import { getPosition, hasSamePosition } from '../position/position';
+
+/**
+ * Graph Vertex Edge
+ */
+export const getAdjacentVertex = (graph: IPosition[], selected: IPosition): IPosition[] => {
```

Comment:
```suggestion
export const getAdjacentVertex = <E extends IPosition>(graph: E[], selected: E): E[] => {
```

Reply:
this way TS know that graph and selected extend IPosition and both have position property